src/webui/api/synccontroller.cpp

```diff
@@ -445,6 +445,14 @@ void SyncController::maindataAction()
 
     data["categories"] = categories;
 
+    QVariantList tags;
+    const auto &tagsList = session->tags();
+    for (auto it = tagsList.cbegin(); it != tagsList.cend(); ++it) {
```

Comment:
pls use range-based for loop here and other places.
`for (const auto &tag : asConst(session->tags()))`